### PR TITLE
Created Activities page

### PR DESCRIPTION
**Pull request by erincuenco**
milestone4-activitiespage

### CODE REVIEW

**Review by rbaguila — COMMENTED** [review]
Assuming that this is just a static page. This PR is okay.

File: src/views/Activities/Activities.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { Container, Header, Item, Icon, Button, Label } from 'semantic-ui-react';
+
```
Comment (rbaguila) [inline src/views/Activities/Activities.js:3]: Add a comment TO-DO for the details page per activity and pagination to indicate your technical debts.

Comment (rbaguila) [reply src/views/Activities/Activities.js:3]: For example, add this at the start of this code:
`// TO-DO: Details page and Pagination`

**Review by rbaguila — APPROVED** [review]

**Comment by rbaguila** [comment]
@erincuenco just take note of my comments. This is approved. 👍

File: src/views/Activities/Activities.js (hunk 1 above)
Comment (erincuenco) [reply src/views/Activities/Activities.js:3]: Noted! Thank you so much! :+1:

**Pull request merged**